meedan/check-web
# Fix/CV2-4064: pagination displaying incorrect list within item navigation

## Description

Do not delete the query params for the 'imported' and 'unmatched' lists
The media page query for items from the unmatched media list was missing the "unmatched":["1"] param, and for the imported lists, it was missing the channel param. This occurred because the simplifyQuery function was removing these params when building the projectmedia URL.

Reference: CV2-4064

## Type of change


## How has this been tested?
- Steps to test from the ticket:
- On any workspace, go to the imported list.
- Observe the total number of items within the list.
- Observe the first and second imported reports in the list view.
- Click on the first item from step 3.
- Observe the number of items next to the pagination.
- Navigate to the next item within the item from step 4 using pagination.

Repeat the steps with the "unmatched" list

## Checklist
